### PR TITLE
Stl 1721 qa overview link and title

### PR DESCRIPTION
**Pull request by SannyNguyenHung**
# Short Description
- Add plural title to no pauschbetrag person A & add overview button

# Feedback
- Any

# How to review this Pull Request
[Link to Confluence](https://digitalservice4germany.atlassian.net/wiki/spaces/STL/pages/117637157/Development+Process#Approve-or-comment) (internal)

### CODE REVIEW

File: webapp/client/src/components/StepNavButtons.js [hunk 1]
```diff
@@ -60,6 +60,44 @@ const Link = styled.a`
...
+  ${sharedButtonLinkStyle}
+
+  padding: 1rem 1.25rem calc(1rem - 4px) 1.25rem; /* The calculation subtracts the border-bottom height. We need a border-bottom for the focus state. */
+
+  font-weight: var(--font-bold);
+  letter-spacing: var(--tracking-wide);
+  text-decoration: none;
+  color: var(--link-color);
+
+  background: white;
```
Comment (punknoir101) [inline webapp/client/src/components/StepNavButtons.js:73]: There is a variable --bg-white for that (although it wouldn't be necessary to have vars for black and white I think)

File: webapp/client/src/components/StepNavButtons.js [hunk 2]
```diff
@@ -60,6 +60,44 @@ const Link = styled.a`
...
+  background-clip: padding-box;
+
+  border: 0;
+  border-radius: 0;
+  border: 1px solid var(--border-color);
```
Comment (punknoir101) [inline webapp/client/src/components/StepNavButtons.js:78]: Why we need this double rules here (border)?

Comment (SannyNguyenHung) [reply webapp/client/src/components/StepNavButtons.js:78]: I added the ${sharedButtonLinkStyle} to the outline link - it should look exactly like the OutlineButton

File: webapp/app/forms/steps/lotse/no_pauschbetrag.py [hunk 3]
```diff
@@ -76,21 +76,17 @@ class StepNoPauschbetragPersonA(LotseFormSteuerlotseStep):
...
                                                num=num_of_users)
-
-    def render(self):
         props_dict = NoPauschbetragProps(
             step_header={
-                'title': str(self.title),
+                'title': str(step_title),
             },
+            showOverviewButton=bool(self.render_info.overview_url),
+            overviewUrl=self.render_info.overview_url,
```
Comment (JulianRoesner) [inline webapp/app/forms/steps/lotse/no_pauschbetrag.py:89]: We could also only pass the overviewUrl and then just check for that in the component. However, I am fine with the more explicit way

File: webapp/app/forms/steps/lotse/no_pauschbetrag.py [hunk 4]
```diff
@@ -76,21 +76,17 @@ class StepNoPauschbetragPersonA(LotseFormSteuerlotseStep):
...
-        self.title = ngettext('form.lotse.no_pauschbetrag.person_a.title',
+        step_title = ngettext('form.lotse.no_pauschbetrag.person_a.title',
                                                'form.lotse.no_pauschbetrag.person_a.title',
                                                num=num_of_users)
-
-    def render(self):
         props_dict = NoPauschbetragProps(
             step_header={
-                'title': str(self.title),
+                'title': str(step_title),
```
Comment (JulianRoesner) [inline webapp/app/forms/steps/lotse/no_pauschbetrag.py:86]: Why did you change that and got rid of the se_multiple_texts method?

File: webapp/client/src/components/StepNavButtons.js (hunk 2 above)
Comment (SannyNguyenHung) [reply webapp/client/src/components/StepNavButtons.js:78]: The border information is overwritten for hover, focus etc. That is why I have to set the borders multiple times. Or do you mean something else?

File: webapp/client/src/components/StepNavButtons.js (hunk 1 above)
Comment (SannyNguyenHung) [reply webapp/client/src/components/StepNavButtons.js:73]: Thanks! :+1: :grin:

File: webapp/client/src/components/StepNavButtons.js (hunk 2 above)
Comment (JulianRoesner) [reply webapp/client/src/components/StepNavButtons.js:76]: Thanks for pointing that out, @punknoir101. I think this is a mistake from the OutlineButton styling and we can delete `border: 0; ` here as well as for the OutlineButton

**Review by JulianRoesner — COMMENTED** [review]
I am not so sure about the changes to the step titles. The rest looks good to me, if we add the changes recommended by @punknoir101 and the tests run through

File: webapp/app/forms/steps/lotse/no_pauschbetrag.py (hunk 4 above)
Comment (SannyNguyenHung) [reply webapp/app/forms/steps/lotse/no_pauschbetrag.py]: Changed it back :grin:

File: webapp/client/src/components/StepNavButtons.js (hunk 2 above)
Comment (SannyNguyenHung) [reply webapp/client/src/components/StepNavButtons.js:76]: Done!

**Review by JulianRoesner — APPROVED** [review]

File: webapp/app/forms/steps/lotse/no_pauschbetrag.py (hunk 3 above)
Comment (SannyNguyenHung) [reply webapp/app/forms/steps/lotse/no_pauschbetrag.py]: I think making it more explicit makes sense here because the parameter for link_overview is a boolean. Regardless of whether a URL was given, the parameter should be checked and not just the URL.
